ssl/ssl_conf.c

```diff
@@ -70,6 +76,12 @@ typedef struct {
 #define SSL_FLAG_VFY_SRV(str, flag) \
         {str, (int)(sizeof(str) - 1), SSL_TFLAG_VFY | SSL_TFLAG_SERVER, flag}
 
+#define SSL_FLAG_CERTVAL(str, flag) \
+    {str, (int)(sizeof(str) - 1), SSL_TFLAG_CERTVAL | SSL_TFLAG_BOTH, flag}
```

Comment:
Spacing does not match surrounding lines, but making it match results in code style errors.